submissions/busolaAdedoyin/javascript/index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>SignUp Page</title>
+    <link href="../css/index.css" type="text/css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Arimo&display=swap" rel="stylesheet">
+    <script src="index.js"></script>
+</head>
+<body>
+
```

Comment:
From line 12 - 105 is irrelevant for this task. All that you needed to do here is link the js file

Reply:
also, there are no closing `body` and `html` tags here

Reply:
> From line 12 - 105 is irrelevant for this task. All that you needed to do here is link the js file

@adaugochi changes have been made and images deleted